cns/service/main.go

```diff
@@ -1002,6 +1005,19 @@ func main() {
 		}(privateEndpoint, infravnet, nodeID)
 	}
 
+	// Check if channel mode is SWIFT V2 and not Direct. And Supply the MAC address to the HNS
+	if cnsconfig.EnableSwiftV2 && cnsconfig.ChannelMode != cns.Direct {
```

Comment:
can we call this function/block somewhere where we already check for cnsconfig & channelmode = CRD/AKS?